setup/README.md

```diff
@@ -12,7 +12,8 @@ main commands to manage ILIAS installations:
 
 `install` and `update` also supply switches and options for a granular control of the inclusion of plugins:
 
-* `--skip <plugin name>` will exclude the named plugin from the command
+* `--skip 
+* There are also named objectives for **import** and **export**. <plugin name>` will exclude the named plugin from the command
```

Comment:
Line broken at wrong offset, I think. `--skip` and ` <plugin name>` probably should be kept together.